batch/batches/Delete/Engines/KDeletingEngine.php

```diff
@@ -74,8 +95,22 @@ public function configure($partnerId, $jobData)
 		$this->pager = new KalturaFilterPager();
 		$this->pager->pageSize = 100;
 		
+		$this->deleteOperationBulkSize = self::DELETE_BACKOFF_BULK_SIZE;
+		if(KBatchBase::$taskConfig->params && KBatchBase::$taskConfig->params->bulkSize)
```

Comment:
We will get php notice if not exist
maybe do
$this->deleteOperationBulkSize = isset(KBatchBase::$taskConfig->params->bulkSize) ? KBatchBase::$taskConfig->params->bulkSize : self::DELETE_BACKOFF_BULK_SIZE;

Reply:
This remains aligned with how other flag checks in batch work so this is why I wanted to keep the code style the same.